core/field_image.js

```diff
@@ -124,6 +139,13 @@ Blockly.FieldImage.prototype.EDITABLE = false;
  */
 Blockly.FieldImage.prototype.isDirty_ = false;
 
+/**
+ *
+ * @type {boolean}
+ * @private
+ */
+Blockly.FieldImage.prototype.flipRtl_ = false;
```

Comment:
Move to the constructor, and add description.

Reply:
Fixed, probably. See [comment on checkbox pr](https://github.com/google/blockly/pull/2876#discussion_r316258252). (I put this one more towards the bottom of the constructor).